### PR TITLE
Remove `UseCase` and rename `TestTag` to `Label`

### PR DESCRIPTION
**Pull request by Frawless**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,12 @@ Plugin itself then parse the data from the annotations and generates `Markdown`
...
 * `@SuiteDoc` - is the main annotation, which consists of all other annotation and should be used right above the method.
-  It contains three fields - `description`, `steps`, and `usecases`, that are set using other annotations.
+  It contains three fields - `description`, `steps`, and `labels`, that are set using other annotations.
 * `@Desc` - overall description of the test, it can contain anything.
 * `@Contact` - contact info with fields `name` and `email`.
 * `@Step` - particular step done in a test, contains two fields - `value` that contains the step information, `expected`
   is for the expected result of the step.
-* `@Usecase` - one of the use-cases that the test is testing.
-* `@TestTag` - specific tag for the test class.
+* `@Labels` - specific label for the test class.
```
Comment (im-konge) [inline README.md:25]:
```suggestion
* `@Label` - specific label for the test class.
```
also the test class is a bit misleading, or? It's not label for the class, but for some area/profile/group of tests, right?

File: README.md [hunk 2]
```diff
@@ -38,12 +37,9 @@ Example of how the test can be annotated:
...
                 @Step(value = "Delete uber operator", expected = "Uber operator is deleted")
         },
-        useCases = {
-                @UseCase(id = "core")
-        },
-        tags = {
-                @TestTag(value = "regression"),
-                @TestTag(value = "clients")
+        labels = {
+                @Label(value = "regression"),
```
Comment (im-konge) [inline README.md:41]: Should this be here? I mean, what area is `regression`? I would add there example like `security`, `auth`, `upgrade`, whatever.

File: README.md [hunk 3]
```diff
@@ -56,13 +52,12 @@ Example of how the test can be annotated:
...
 * `@TestDoc` - is the main annotation, which consists all other annotation and should be used right above the method.
-  It contains three fields - `description`, `steps`, and `usecases`, that are set using other annotations.
+  It contains three fields - `description`, `steps`, and `labels`, that are set using other annotations.
 * `@Desc` - overall description of the test, it can contain anything.
 * `@Contact` - contact info with fields `name` and `email`.
 * `@Step` - particular step done in a test, contains two fields - `value` that contains the step information, `expected`
   is for the expected result of the step.
-* `@Usecase` - one of the use-cases that the test is testing.
-* `@TestTag` - specific tag for the test class.
+* `@Label` - specific label for the test class.
```
Comment (im-konge) [inline README.md:60]: Same as the previous comment.

File: README.md [hunk 4]
```diff
@@ -75,14 +70,9 @@ Example of how the test can be annotated:
...
-                @UseCase(id = "core"),
-                @UseCase(id = "core+"),
-                @UseCase(id = "core+++")
-        },
-        tags = {
-                @TestTag(value = "default"),
-                @TestTag(value = "regression"),
+        labels = {
+                @Label(value = "default"),
+                @Label(value = "regression"),
```
Comment (im-konge) [inline README.md:75]: Same here

File: test-docs-generator-maven-plugin/src/main/java/io/skodjob/MdGenerator.java [hunk 5]
```diff
@@ -136,37 +137,37 @@ public static void createTestRecord(PrintWriter write, TestDoc testDoc, String c
...
-        if (testDoc.tags().length != 0) {
-            write.println(TextStyle.boldText("Tags:"));
-            write.println();
-            write.println(TextList.createUnorderedList(createTags(testDoc.tags())));
-        }
+//        if (testDoc.labels().length != 0) {
+//            write.println(TextStyle.boldText("Tags:"));
+//            write.println();
+//            write.println(TextList.createUnorderedList(createLabels(testDoc.labels())));
+//        }
```
Comment (im-konge) [inline test-docs-generator-maven-plugin/src/main/java/io/skodjob/MdGenerator.java:162]: This should be here?

**Review by im-konge — APPROVED** [review]

**Review by kornys — APPROVED** [review]

**Pull request merged**